namd/src/colvarproxy_namd.C

```diff
@@ -1466,6 +1466,15 @@ int colvarproxy_namd::compute_volmap(int flags,
 
 #if CMK_SMP && USE_CKLOOP // SMP only
 
+int colvarproxy_namd::check_smp_enabled()
```

Comment:
Ok. Did I misinterpret [this comment](https://github.com/Colvars/colvars/pull/338#discussion_r402554373)?

Reply:
Can you please do both cases, so that we can use the same runtime to test SMP and non-SMP code path?

Also, your commit will break NAMD compilation for multicore builds (the GH tests pass only because they are building NAMD without SMP)